### DISCUSSION TITLE
wakeups/second

### DISCUSSION THREAD

**Discussion by roumano** [Koenkk/zigbee2mqtt#6946] [General]
Hi,
zigbee2mqtt working very fine for me, so issue.

But i wondering if it's normal if zigbee2mqtt consume always ~5% of CPU (on a Pi4).

When doing a strace of the PID, i see something like 3000 clock_gettime per second( like `clock_gettime(CLOCK_MONOTONIC, {tv_sec=125521, tv_nsec=997610561}) = 0`

a powertop say arround 300 events per second (when zigbee2mqtt is idle )

Debug mode don't provide any usefull information for these kind of wakeup

**Comment by roumano** [comment]
can someone also try with a `powertop` command if zigbee2mqtt is making lot of Events/s ?
To confirm if it's happen to only me or to everyone.
Tkx

**Reply by canton7** [reply]
I'm seeing about 250 events/sec from zigbee2mqtt as well, costing me a full watt.
